core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalWriter.java

```diff
@@ -40,41 +39,33 @@
 public class RaftJournalWriter implements JournalWriter {
   private static final Logger LOG = LoggerFactory.getLogger(RaftJournalWriter.class);
   // How long to wait for a response from the cluster before giving up and trying again.
-  private final long mWriteTimeoutMs;
-  private final long mEntrySizeMax;
-  private final long mFlushBatchBytes;
+  private final long mWriteTimeoutMs =
+      ServerConfiguration.getMs(PropertyKey.MASTER_EMBEDDED_JOURNAL_WRITE_TIMEOUT);
+  // journal entry size max is the hard limit set by underlying ratis
```

Comment:
I'm wondering if these should even be variables, and instead should just access the property key directly?

Reply:
Fixed.